### PR TITLE
express-server

### PR DESCRIPTION
**Pull request by Zhenyatta**
Hope this one is a correct pull request :)

### CODE REVIEW

File: back-end/app.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const express = require("express");
```
Comment (RobMin) [inline back-end/app.js:1]: Pleas import using `import` instead of `require`.

**Review by RobMin — CHANGES_REQUESTED** [review]
Overall LGTM, but let's also change the names of folders and files please.

Frontend and backend are commonly used both ways and single worded dir names are more comfortable to use so IMO it's better to not use dashes.
front-end -> frontend
back-end -> backend

Let's also change `app.js` to `index.js`.
Usually in other languages `main` is the starting file of the project, in html, css, js it's usually `index` e.g. if you host a directory from backend lets say `/mydir` the browsers will open `index.html` and display it right away if it exists.

**Review by RobMin — CHANGES_REQUESTED** [review]
Access test

File: back-end/app.js (hunk 1 above)
Comment (Zhenyatta) [reply back-end/app.js:1]: it throws an error :(

Comment (Zhenyatta) [reply back-end/app.js:1]: lav dzvec

**Pull request merged**